packages/vaadin-lumo-styles/components/charts.css

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+@import '../src/mixins/base-layer-reset.css' vaadin-chart;
```

Comment:
IMO ideally we shouldn't need to reset base styles for chart, and only have Lumo define custom CSS properties.

Reply:
@jouni what do you think? Do you plan to modify anything beyond the chart CSS properties in the base theme?